neo/public/javascripts/common/services.js

```diff
@@ -74,6 +74,18 @@ angular.module('cloudberry.common', [])
       ];
     }
 
+    function getPopulationTarget(parameters){
+      var targetLowerCase = parameters.geoLevel;
+      var targetUpperCase = targetLowerCase.charAt(0).toUpperCase() + targetLowerCase.substr(1);
+      return {
+        joinKey: [targetLowerCase],
+        dataset: "twitter.ds" + targetUpperCase + "Population",
```

Comment:
It's better to have some meta object to map state/county/city to its dataset. Try to avoid string concatenation...